netlify.toml

```diff
@@ -10,7 +10,7 @@
   command = """\
   cd ../
   rm -rf monodocs
-  git clone https://github.com/apollographql/docs --branch release/1.0 --single-branch monodocs
+  git clone https://github.com/apollographql/docs --branch main --single-branch monodocs
```

Comment:
Is this targeting of `main` instead of `release/1.0` correct? cc. @trevorblades

Reply:
Yeah, I made the same mistake here. This is cloning the docs repo not ours!